### PR TITLE
cups{accept,reject,enable,disable}: add pages; lpstat: add pt_BR translation

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_BR/common/cupsdisable.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# cupsdisable
+
+> Stop printers and classes.
+> See also: `cupsenable`, `cupsaccept`, `cupsreject`, `lpstat`.
+> More information: <https://www.cups.org/doc/man-cupsenable.html>.
+
+- Stop one or more destination(s):
```
Comment (gutjuri) [inline pages.pt_BR/common/cupsdisable.md:7]: What is a destination? A printer?

File: pages.pt_BR/common/cupsdisable.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+`cupsdisable {destination1 destination2 ...}`
```
Comment (gutjuri) [inline pages.pt_BR/common/cupsdisable.md:9]:
```suggestion
`cupsdisable {{destination1 destination2 ...}}`
```

File: pages/common/cupsaccept.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# cupsaccept
+
+> Accept jobs sent to one or more destinations.
```
Comment (gutjuri) [inline pages/common/cupsaccept.md:3]: Again, please specify what a destination is.

File: pages/common/cupsdisable.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# cupsdisable
+
+> Stop printers and classes.
+> See also: `cupsenable`, `cupsaccept`, `cupsreject`, `lpstat`.
+> More information: <https://www.cups.org/doc/man-cupsenable.html>.
+
+- Stop one or more destination(s):
+
+`cupsdisable {destination1 destination2 ...}`
```
Comment (gutjuri) [inline pages/common/cupsdisable.md:9]:
```suggestion
`cupsdisable {{destination1 destination2 ...}}`
```

File: pages.pt_BR/common/cupsdisable.md
Comment (gutjuri) [inline pages.pt_BR/common/cupsdisable.md:1]: This page is in english :P

File: pages.pt_BR/common/cupsaccept.md
Comment (gutjuri) [inline pages.pt_BR/common/cupsaccept.md:1]: This is in the pt_BR directory even though the page is in english

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks for your contribution, I have some comments.

File: pages.pt_BR/common/cupsdisable.md (hunk 1 above)
Comment (vitorhcl) [reply pages.pt_BR/common/cupsdisable.md:7]: > What is a destination? A printer?

@gutjuri a printer or a class of printers.

File: pages.pt_BR/common/cupsdisable.md
Comment (vitorhcl) [reply pages.pt_BR/common/cupsdisable.md:1]: Sorry, I messed things up :P

**Review by kbdharun — APPROVED** [review]

File: pages.pt_BR/common/cupsdisable.md (hunk 1 above)
Comment (gutjuri) [reply pages.pt_BR/common/cupsdisable.md:7]: In my opinion, this should be clarified in the page.

File: pages/common/cupsdisable.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+> Veja também: `cupsenable`, `cupsaccept`, `cupsreject`, `lpstat`.
+> Mais informações: <https://www.cups.org/doc/man-cupsenable.html>.
+
+- Para um ou mais destino(s):
+
+`cupsdisable {{destino1 destino2 ...}}`
+
+- Cancela todos os trabalhos do(s) destino(s) especificado(s):
+
+`cupsdisable -c {{destination1 destination2 ...}}`
```
Comment (isaacvicente) [inline pages/common/cupsdisable.md:13]:
```suggestion
`cupsdisable -c {{destino1 destino2 ...}}`
```

File: pages.pt_BR/common/cupsdisable.md (hunk 1 above)
Comment (isaacvicente) [reply pages.pt_BR/common/cupsdisable.md:7]: @gutjuri I think we could use `{{printer|class}}`, right?

File: pages.pt_BR/common/cupsaccept.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+> Aceita trabalhos enviados para um ou mais destinos.
+> Veja também: `cupsreject`, `cupsenable`, `cupsdisable`, `lpstat`.
+> Mais informações: <https://openprinting.github.io/cups/doc/man-cupsaccept.html>.
+
+- Aceita trabalhos de impressão para os destinos especificados:
+
+`cupsaccept {{destination1 destination2 ...}}`
+
+- Especificar um servidor diferente:
```
Comment (isaacvicente) [inline pages.pt_BR/common/cupsaccept.md:11]:
```suggestion
- Especifica um servidor diferente:
```

File: pages.pt_BR/common/cupsaccept.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+`cupsaccept -h {{server}} {{destination1 destination2 ...}}`
```
Comment (isaacvicente) [inline pages.pt_BR/common/cupsaccept.md:13]:
```suggestion
`cupsaccept -h {{server}} {{destino1 destino2 ...}}`
```

File: pages.pt_BR/common/cupsdisable.md (hunk 1 above)
Comment (isaacvicente) [reply pages.pt_BR/common/cupsdisable.md:7]: As "destination" is used all over the place, could we put a NOTE in the beginning of the command description?
Like: `NOTE: destination is referred as a printer or a class of printers`

What do you guys think @gutjuri @vitorhcl ?

Comment (isaacvicente) [reply pages.pt_BR/common/cupsdisable.md:7]: I think `{{printer|class}}` syntax would pollute the descriptions of a bunch of commands.

**Review by isaacvicente — COMMENTED** [review]
Fiz sugestões sobre o idioma usado. `destino` é o que queremos em vez de `destination`.
(talvez tu tenha esquecido de atualizar e traduzir pra português também)

**Review by isaacvicente — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Pull request merged**